frontend/src/metabase/entities/users/forms.js

```diff
@@ -13,12 +13,14 @@ const getNameFields = () => [
     title: t`First name`,
     placeholder: "Johnny",
     autoFocus: true,
+    // XXX: Make the name optional after BE has relaxed name field constraint.
```

Comment:
I left this in place, so I could be easily followed up in another PR, let me know if this shouldn't be here.